src/scripts/Update.ts

```diff
@@ -800,6 +800,16 @@ class Update implements Saveable {
 
         '0.9.8': ({ playerData, saveData }) => {
             saveData.oakItemLoadouts = saveData.oakItemLoadouts.map((list, index) => ({ name: `Loadout ${index + 1}`, loadout: list }));
+            //UNTESTED POKERUS UPDATE FUNCTION - MAKE SURE THIS WORKS!!!!
+            saveData.party.caughtPokemon.forEach(p => {
+                if (p[8]) {
+                    delete p[8];
+                    p[8] = GameConstants.Pokerus.Contagious;
+                } else {
+                    delete p[8];
+                    p[8] = GameConstants.Pokerus.None;
```

Comment:
Maybe ternary is more readable ? 
```suggestion
              delete p[8] ;
              p[8] = (p[8]) ? Game Constants.Pokerus.Contagious : GameConstant.Pokerus.None;
```

Reply:
If you do that you definitely shouldn't delete the value before you read it.

Reply:
Either way, I believe we usually avoid using external constants or enums inside update scripts because they might change in the future.

Reply:
Made it ternary and removed constants, thanks! 👍